src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int upperBase;
+    private final int lowerBase;
+    private final int height;
+
+    public IsoscelesTrapezoid(int upperBase, int lowerBase, int height, String color) {
+        super(color);
+        this.upperBase = upperBase;
+        this.lowerBase = lowerBase;
+        this.height = height;
+    }
+
+    public int getUpperBase() {
+        return upperBase;
+    }
+
+    public int getLowerBase() {
+        return lowerBase;
+    }
+
+    public int getHeight() {
+        return height;
+    }
+
+    @Override
+    public double getArea() {
+        return (upperBase + lowerBase) * height / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: "
+                + getArea()
+                + " sq.units, upper base: "
+                + upperBase
+                + " units, lower base: "
+                + lowerBase
+                + " units, height: "
+                + height
+                + " units, color: "
+                + getColor());
```

Comment:
let's make this a bit more concise
```suggestion
                + getArea() + " sq.units, upper base: "  + upperBase
                + " units, lower base: " + lowerBase
                + " units, height: " + height
                + " units, color: " + getColor());
```
please fix in the same way for `draw()` method in other classes.